OpenRA.Mods.Common/Traits/Buildings/TransformsIntoMobile.cs

```diff
@@ -111,7 +111,7 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 				if (!order.Queued && activity.NextActivity != null)
 					activity.NextActivity.Cancel(self);
 
-				activity.Queue(self, new IssueOrderAfterTransform("Move", order.Target));
+				activity.Queue(new IssueOrderAfterTransform("Move", order.Target));
```

Comment:
This is in the wrong commit ("Make Tick return bool." instead of "Move ChildActivity handling into base Activity class.").